spec/controllers/admin/facility_groups_controller_spec.rb

```diff
@@ -241,31 +178,8 @@
     end
   end
 
-  describe "DELETE #destroy with regions prep disabled" do
```

Comment:
This was a duplicated block with and without `regions_prep`, now everything is for enabled in the original block.